### DISCUSSION TITLE
TAG-3.3.1

### DISCUSSION THREAD

**Discussion by Tatsu-syo** [Tatsu-syo/noMeiryoUI#112] [General]
- アプリケーションフォントを選択して何らかのシステムフォントを変更するとタイトルバーの表示が消える不具合を修正しました。
- 簡体字中国語(中国)の訳を更新しました。(zh-cn.lng)kingofotaku様に感謝します。
- 繁体字中国語(台湾)の訳を更新しました。(zh-TW.lng)abc0922001様に感謝します。

----

- Fix bug when application font and system fonts, application title disappeared..
- Improve Simplified Chinese(China) translations.Thanks for kingofotaku.(zh-CN.lng and help contents.)
- Improve Traditional Chinese(Taiwan) translations.Thanks for abc0922001.(zh-TW.lng)

<hr /><em>This discussion was created from the release <a href='https://github.com/Tatsu-syo/noMeiryoUI/releases/tag/TAG-3.3.1'>TAG-3.3.1</a>.</em>

**Comment by seaclub1** [comment]
windows10 x64    10.0.19045.4046

On the desktop or navigator
Is it difficult to change the context-menu font that pops up when the mouse right-click comes up?

![0307-003](https://github.com/Tatsu-syo/noMeiryoUI/assets/162572202/dc4947e8-9c06-4fa3-87ef-944573e9452f)
![0307-001](https://github.com/Tatsu-syo/noMeiryoUI/assets/162572202/f7944b19-c42b-42fa-a305-a3beea27d7a7)
![0307-002](https://github.com/Tatsu-syo/noMeiryoUI/assets/162572202/2774b4b2-5965-4c78-ae83-1336a672ff03)

**Comment by Tatsu-syo** [comment]
I think system font is ignored desktop context menu.
Since Windows 10, Windows and many programs made by Microsoft ignore system fonts.
What is system fonts?
Especially in Japan, Windows 8.x and Windows 10/11 use very ugly font.
I'm very sad.

**Reply by nellyfurtado** [reply]
We east-asia care about writing and font

**Reply by Tatsu-syo** [reply]
I also.
But Microsoft, don't think so. Especially in Japan.
Microsoft employees in Japan appear to be doing nothing.

**Comment by 17637501804** [comment]
> * アプリケーションフォントを選択して何らかのシステムフォントを変更するとタイトルバーの表示が消える不具合を修正しました。
> * 簡体字中国語(中国)の訳を更新しました。（zh-cn.lng）Kingofotaku様に感謝します。
> * 繁体字中国語(台湾)の訳を更新しました。（zh-TW.lng）abc0922001様に感謝します。
> 
> * 修复应用程序字体和系统字体、应用程序标题消失的bug..
> * 改进简体中文（中国）翻译。感谢 kingofotaku。（zh-CN.lng 和帮助内容。
> * 改进繁体中文（台湾）翻译。感谢 abc0922001。（zh-TW.lng）
> 
> _此讨论创建自[版本 TAG-3.3.1](https://github.com/Tatsu-syo/noMeiryoUI/releases/tag/TAG-3.3.1)。_
こんにちは管理者様、この問題について報告したいのですが、3.31バージョンを使用すると、フォントが変更された後にシステムフォントが縦書きになる現象が見られます。例えば、24h2システムのメモ帳を開くと、フォントが縦書きで配置されており、その結果、メモ帳が「メモリ不足」のエラーを報告して使用できません。しかし、メモ帳をアンインストールすると問題は解消されます。また、一部の動画アプリの説明文フォントも同様に縦書き表示されます。この問題をどのように解決できるか教えていただけますか？

**Reply by Tatsu-syo** [reply]
@17637501804 一旦システムフォントをプリセットに戻して再度、設定してみるのはいかがでしょうか。
また、縦書きの場合はフォント名の頭に @ がつきますが、あやまって@をつけたということはないでしょうか。
これで改善しないようであればフォントキャッシュを削除してから再起動するとどうなるでしょうか。
よろしくお願いします。

**Comment by 17637501804** [comment]
このソフトを使っているときに、同じような問題が発生したことはありますか？
今夜、一度アンインストールして再インストールしてみようと思います。
それと、「@」についてはよく分からないのですが、すべてのフォントを変更した後にメモ帳やいくつかの動画サイトで説明文を確認すると、文字が縦書きのように数列の形で表示されてしまいます。

---Original---
From: "Tatsuhiko ***@***.***&gt;
Date: Mon, Dec 2, 2024 09:23 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Tatsu-syo/noMeiryoUI] TAG-3.3.1 (Discussion #112)




 
@17637501804 一旦システムフォントをプリセットに戻して再度、設定してみるのはいかがでしょうか。
 また、縦書きの場合はフォント名の頭に @ がつきますが、あやまって@をつけたということはないでしょうか。
 これで改善しないようであればフォントキャッシュを削除してから再起動するとどうなるでしょうか。
 よろしくお願いします。

**Comment by fanwu89** [comment]
Edit: I saw the answer, Thanks! 

Tatsu-syo on Mar 7, 2024
"I think system font is ignored desktop context menu.
Since Windows 10, Windows and many programs made by Microsoft ignore system fonts.
What is system fonts?
Especially in Japan, Windows 8.x and Windows 10/11 use very ugly font.
I'm very sad."

--------
Hi! Tatsu-syo
The Explorer menu fonts has not changed, but other softwares can work.
Windows 10 Pro 22H2 19045.5737 Simplified Chinese

![desktop_menu](https://github.com/user-attachments/assets/d6bdc248-a472-4802-88fa-a6e31b05b178)

![explorer_menu](https://github.com/user-attachments/assets/62b3ae69-e9c0-45f9-9e1a-ccc77101b10b)

![fb2k_menu](https://github.com/user-attachments/assets/5392e6b4-574c-47a9-83ce-28a1d243b63f)

![tc_menu](https://github.com/user-attachments/assets/b4b18c6a-2cec-4ae4-b6f8-ae4fdf6d0a0d)

**Comment by Tatsu-syo** [comment]
It's a known fact. Please read [FAQ](https://github.com/Tatsu-syo/noMeiryoUI/wiki/FAQ).
Since Windows 10, Microsoft ignore system font settings and enforce their fonts design.
Especially Japanese version uses very ugly font.

**Reply by fanwu89** [reply]
Thank you for the answer. 
I had to upgrade to Windows 10 to use new softwares, but I still prefer the old system (ex. Windows XP, Windows 7), pixel monospaced fonts.
